front/components/assistant/conversation/AgentMessage.tsx

```diff
@@ -184,7 +184,7 @@ export function AgentMessage({
     switch (event.type) {
       case "agent_action_success":
         setStreamedAgentMessage((m) => {
-          return { ...updateMessageWithAction(m, event.action), content: "" };
+          return { ...updateMessageWithAction(m, event.action) };
```

Comment:
We no longer want to clear the message body when we receive an action, since we can update the body in multiple steps.